Task4.js

```diff
@@ -0,0 +1,11 @@
+// Format any integer provided into a string with "," (commas) in the correct places.
+
+// Example:
+
+// For n = 100000 the function should return '100,000';
+// For n = 5678545 the function should return '5,678,545';
+// for n = -420902 the function should return '-420,902'.
+
+var numberFormat = function (number) {
```

Comment:
you can improve the code base by making this an arrow function and by declaring it using const instead of var